spacy_experimental/biaffine_parser/pytorch_bilinear.py

```diff
@@ -2,6 +2,51 @@
 from torch import nn
 
 
+class Bilinear(nn.Module):
```

Comment:
Wondering if it makes sense to also include a test or two for this module.

Reply:
I guess that we could test it against the PyTorch implementation (copying over parameters).

Reply:
Done.